package.json

```diff
@@ -392,6 +392,7 @@
     "embedding-sdk:generate-package": "node ./bin/embedding-sdk/generate-sdk-package-files.js",
     "embedding-sdk:generate-changelog": "yarn conventional-changelog --config enterprise/frontend/src/embedding-sdk/conventional-changelog-config.js",
     "embedding-sdk:publish": "cd ./resources/embedding-sdk && npm publish",
+    "embedding-sdk:generate-nextjs-compat": "npx tsx enterprise/frontend/src/embedding-sdk/bin/generate-nextjs-compat.ts",
```

Comment:
This will not work in ci, I asked on the FE channel if we can install tsx as dev dep: https://metaboat.slack.com/archives/C505ZNNH4/p1733487433776769